docker/run-jetmon.sh

```diff
@@ -19,4 +19,5 @@ if [ ! -f config/db-config.conf ]; then
 	sed -e "s/<MYSQLDB_USER>/${MYSQLDB_USER}/g" -e "s/<MYSQLDB_ROOT_PASSWORD>/${MYSQLDB_ROOT_PASSWORD}/g" -e "s/<MYSQLDB_PORT>/${MYSQLDB_DOCKER_PORT}/g" -e "s/<MYSQLDB_DATABASE>/${MYSQLDB_DATABASE}/g" config/db-config-sample.conf > config/db-config.conf
 fi
 
+npm install
```

Comment:
Do we need this?
We already have `npm install` inside **Dockerfile_jetmon**

Reply:
Oh, missed that entry. I wanted to re-run the service without having to rebuild the image. Going to remove it, as it's not a big blocker for the PR and it's done rarely enough.